test/test_images/hellovolume/hellovolume.go

```diff
@@ -0,0 +1,45 @@
+/*
```

Comment:
Not for this PR, but it would be good to have this consolidated with many of the other test images into a single released "tell me all" type binary, where the test would query for e.g. `"/filecontents?path=" + test.HelloVolumePath`.

Reply:
I believe this is something @dgerd has been slowly pursuing